lms/templates/courseware/courseware.html

```diff
@@ -25,13 +25,27 @@
     <%static:include path="common/templates/${template_name}.underscore" />
 </script>
 % endfor
+
 % if settings.FEATURES.get('ENABLE_COURSEWARE_SEARCH'):
     % for template_name in ["course_search_item", "course_search_results", "search_loading", "search_error"]:
-    <script type="text/template" id="${template_name}-tpl">
-        <%static:include path="search/${template_name}.underscore" />
-    </script>
-% endfor
+        <script type="text/template" id="${template_name}-tpl">
+            <%static:include path="search/${template_name}.underscore" />
+        </script>
+    % endfor
 % endif
+
+% for template_name in ["proctored-exam-status"]:
+  <script type="text/template" id="${template_name}-tpl">
+      <%static:include path="courseware/${template_name}.underscore" />
+  </script>
+% endfor
+
+% for template_name in ["course_search_item", "course_search_results", "search_loading", "search_error"]:
```

Comment:
I wonder if there was some merge conflict that was not properly resolved? We should have not touched anything regarding that course_search template includes